Free Test
Standard Top Level Checks:                 2;  Failures:  0  [Deep checks: 1]

******  Critical Failures:  1  ******


Error -- Unexpected Exception:
Throw after check

Exception thrown after check completed
Last Recorded Message:

Tests/Tests/TestFramework/TestRunnerDiagnosticsUtilities.hpp, Line 39
Phoney equality check

[int]

=======================================

